query.go

```diff
@@ -552,11 +564,22 @@ func (qr *QueryResult) Scan(dest ...interface{}) error {
 					return err
 				}
 				*d = b
+			case bool:
+				*d = src
 			case nil:
 				trace("%s: skipping nil scan data for variable #%d (%s)", qr.conn.ID, n, qr.columns[n])
 			default:
 				return fmt.Errorf("invalid bool col:%d type:%T val:%v", n, src, src)
 			}
+		case *[]uint8:
```

Comment:
I'd suggest using `[]byte` instead of `[]uint8` just to follow conventional Go code. `byte` is basically an alias for `uint8`

Reply:
IIRC, I've tried `[]byte` before, I'm gonna re-check again if `[]byte` works.

Reply:
Ah I see. That's fine then, let's keep `uint8`.

Reply:
Have you tried defining the destination parameter to be a `[]byte` instead of `[]uint8` when calling the `Scan` function? With that being said, I don't mind keeping `[]uint8`.